easycv/hooks/yolox_lr_hook.py

```diff
@@ -4,7 +4,7 @@
                                           annealing_cos)
 
 
-@HOOKS.register_module()
+@HOOKS.register_module(force=True)
```

Comment:
@Cathy0908  check this, any compatibility problem with mmdet?

Reply:
This function is the same as mmdet, so it doesn't matter which one to use